packages/aws-cdk/CONTRIBUTING.md

```diff
@@ -42,6 +42,13 @@ To run regression tests in the source tree:
 $ test/integ/test-cli-regression-against-current-code.sh [-t '...']
 ```
 
+Integ tests can run in parallel across multiple regions. Set the `AWS_REGIONS`
```

Comment:
Can you add a bit more about what the striping strategy will be, or how tests get assigned to regions?

Reply:
Also state which region will be picked by default.

Reply:
> Also state which region will be picked by default.

Why?

Reply:
>>    Also state which region will be picked by default.
>>
> All of them?
> 
> Why do you care? If you care, then you shouldn't be passing all of them.

Not what I meant. Which region will be picked if `AWS_REGIONS` is not specified.

Reply:
I might be good for debugging if each test prints out the region its running in